cli/internal/cache/cache.go

```diff
@@ -65,11 +71,11 @@ func newSyncCache(config *config.Config, remoteOnly bool, recorder analytics.Rec
 		mplex.caches = append(mplex.caches, newHTTPCache(config, recorder))
 	}
 	if len(mplex.caches) == 0 {
-		return nil
+		return nil, ErrNoCachesEnabled
```

Comment:
Upstream addressed this a bit differently: https://github.com/thought-machine/please/commit/e0b5e2900f5caab98a20d476d34bce66da28cdb8

Reply:
`turbo` still gives benefits from the topsort so running without using cache isn't *entirely* nonsensical.

Aside, this is itching for a better UI:
- Don't write cache: `--no-cache`
- Don't read cache: `--force` or `TURBO_FORCE`
- Don't read/write local cache: `--remote-only` or `TURBO_REMOTE_ONLY`
- Don't read/write remote cache: `turbo logout`

Reply:
I'm all for a better set of flags here, including a way to run local-only without logging out (or unlinking)